problem10.py

```diff
@@ -0,0 +1,60 @@
+#The Sieve of Erastothenes
+import math
+from get_number_from_index import get_number_from_index
+
+
+def get_idx_from_number(number):
+    index = (number-3) >> 1
+    return index
+
+
+
+def get_number_from_index(index):
+    number = 2*index + 3
+
+    return number
+
+
+def SieveOfErastothenes(n):
+    n_list = []
+    for i in range(3, n, 2):
+        n_list.append(True)
+
+
+    sum = 2
+    for j in range(len(n_list)):
+
+        if n_list[j] == False:
+            continue
+
+        number =get_number_from_index(j) #get numb from inx
+
+        for k in range(number**2, n, number):
+            n_list[(k-3)>>1] = False #get idx from numb
+
+        if n_list[j] == True:
+            sum+=2*j + 3
+
+    
```

Comment:
Inside one block of code, like a function, please use a maximal 1 new line between code.

Reply:
please change it also in other files.